collector/binlog.go

```diff
@@ -71,6 +78,11 @@ func ScrapeBinlogSize(db *sql.DB, ch chan<- prometheus.Metric) error {
 	ch <- prometheus.MustNewConstMetric(
 		binlogFilesDesc, prometheus.GaugeValue, float64(count),
 	)
+	// The last row contains the last binlog file number.
+	value, _ := strconv.ParseFloat(strings.Split(filename, ".")[1], 64)
+	ch <- prometheus.MustNewConstMetric(
+		binlogFileNumberDesc, prometheus.GaugeValue, value,
```

Comment:
I think it's safe enough to assume this is a Counter.  I think in all cases MySQL will only reset to 0.